src/bin/conserve.rs

```diff
@@ -145,6 +145,9 @@ enum Command {
         /// Show only version names.
         #[structopt(long, short = "q")]
         short: bool,
+        /// Sort bands to show most recent first.
+        #[structopt(long, short = "n", conflicts_with = "short" )]
```

Comment:
conflicts_with = "short"

I think it would be fine to use this together with --short, to change the order?